CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ AppEngine version, listed here to ease deployment and troubleshooting.
 
 ## Next Release (replace with git tag when deployed)
 
+## `20241008t115400-all`
+
```

Comment:
Maybe we should add more changelog entries for smaller items when a release doesn't have a version bump.